src/helpers/build.js

```diff
@@ -70,21 +70,22 @@ const copySupportFiles = folder => {
   if (process.env.LNG_LIVE_RELOAD === 'true' && command === 'dev') {
     const host = process.env.LNG_LIVE_RELOAD_HOST || 'localhost'
     const port = process.env.LNG_LIVE_RELOAD_PORT || 8991
+    const host = process.env.LNG_LIVE_RELOAD_HOST || 'localhost'
```

Comment:
Remove this line as this is duplicate of line 71

Reply:
done